parsers/c/c_parser.mly

```diff
@@ -2071,6 +2071,9 @@ expr_without_let:
 | e1= list_expr QUESTION e2= list_expr COLON e3= list_expr
     { Cerb_frontend.Cn.(CNExpr ( Cerb_location.(region ($startpos, $endpos) (PointCursor $startpos($2)))
                                , CNExpr_ite (e1, e2, e3))) }
+| e1= list_expr EQ_EQ_GT e2= list_expr (* implication arrow ==> *)
```

Comment:
This addition is unnecessary and may be deleted. The correct place to add this would be between `bool_and_expr` and `bool_or_expr` as a `bool_implies_expr`.

Reply:
Should the base case for this be `bool_and_expr` like it is for `bool_or_expr`? Or should the base case be `bool_or_expr`?